clients/general/client.go

```diff
@@ -70,3 +71,9 @@ func (gc *generalRestClient) FetchMetrics(ctx context.Context) (string, error) {
 	body, err := clients.GetRequest(gc.url+clients.ApiMetricsRoute, ctx)
 	return string(body), err
 }
+
+func (gc *generalRestClient) SetConfiguration(service string, key string, value string, ctx context.Context) error {
+	// TODO: At the end of the current scenario, the SMA should be ready to Set Configuration Value for
+	//  the provided  (ServiceKey, Key, Value)
+	return nil
```

Comment:
There's no implementation here...?

Reply:
- Yep, this is planned for a follow-up issue [#1432 - Implement the design: SMA Sets Configuration (Writable) Without Restart .](https://github.com/edgexfoundry/edgex-go/issues/1432)

Reply:
@tsconn23 In light of our discussion, it makes sense to have this work be integration-ready (with the code-base in the `edgex-go` repo). So I'm pulling this in (given that it was _originally_ planned for a follow-up PR) and will be implementing this shortly 👍